bank-sdk/example-app/src/androidTest/java/net/gini/android/bank/sdk/exampleapp/ui/screens/NoResultScreen.kt

```diff
@@ -0,0 +1,94 @@
+package net.gini.android.bank.sdk.exampleapp.ui.screens
+
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.ViewActions.click
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.matcher.ViewMatchers.isDisplayed
+import androidx.test.espresso.matcher.ViewMatchers.withText
+
+class NoResultScreen {
```

Comment:
[Request] Please update the methods in this class to match the changes you made for [this](https://github.com/gini/gini-mobile-android/pull/565) PR. 
By that I mean returning the result of the check to be used in the main test case.